MIT-LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020 Linq To DB
```

Comment:
Not sure how technically accurate this would be, at least in the United States.  Probably just fine as-is.  But unless a legal entity was formed, along with an agreement that contributions would be assigned to the "Linq To DB" entity, it's probably inaccurate.

FYI, in the United States, copyright is granted regardless of whether or not a "Copyright (c) 2020 {name}" notice appears in the work.

So it's not a big deal, although I'm guessing it would be more accurate to have the names of each of the developers here.

Here's another suggestion, although again, it's just a suggestion and I'm not necessarily advocating it:
```suggestion
Copyright (c) 2020 {major contributors}, and others
```
One last thing: I think the year here is supposed to be the year of first publication.  Of course, as this is updated continuously, there would be multiple years of first publication.  This is why it is typical to see something like:
```suggestion
Copyright (c) 2018-2020 Linq To DB
```
See: https://www.copyright.gov/circs/circ03.pdf for even more details about copyright notices in the United States

Reply:
I reviewed the copyright license agreement (CLA) in effect for this project.  It does not transfer copyright, but rather requires contributors to license redistribution and patent rights to the .Net Foundation for this project.  Just FYI.

Reply:
it's too complicated :) https://opensource.stackexchange.com/questions/5778/why-do-licenses-such-as-the-mit-license-specify-a-single-year

will use Copyright © 2016-2020 Igor Tkachev, Ilya Chudin, Svyatoslav Danyliv, Dmitry Lukashenko for now...